sdk/monitor/Azure.Monitor.OpenTelemetry/README.md

```diff
@@ -1,19 +1,53 @@
 # Azure Monitor Distro client library for .NET
 
+The Azure Monitor Distro is a client library that sends telemetry data to Azure Monitor following the OpenTelemetry Specification. This library can be used to instrument your ASP.NET Core applications to collect and send telemetry data to Azure Monitor for analysis and monitoring.
+
 ## Getting started
 
 ### Prerequisites
 
+- **Azure Subscription:**  To use Azure services, including Azure Monitor Distro, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free/dotnet/) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://azure.microsoft.com/account).
+- **Azure Application Insights Connection String:** To send telemetry data to the monitoring service you'll need connection string from Azure Application Insights. If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [Create an Application Insights resource](https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource) and [copy the connection string](https://docs.microsoft.com/azure/azure-monitor/app/sdk-connection-string?tabs=net#find-your-connection-string).
+
 ### Install the package
 
+Install the Azure Monitor Distro for .NET with [NuGet](https://www.nuget.org/):
+
+```dotnetcli
+dotnet add package Azure.Monitor.OpenTelemetry.AspNetCore --prerelease
+```
+
 ### Authenticate the client
```

Comment:
Actually we're doing two things here, right? Configure and Authenticate.
I wonder if we should have basic configure for the easiest way to get started and layer on authentication later.
I don't want customers think they NEED it, esp b/c it's one more point of failure for the simplest way to get started.

Reply:
Updated the PR with getting started app, please check.